ci/fih_test_docker/Dockerfile

```diff
@@ -33,21 +36,12 @@ RUN \
         imgtool==1.6.0 \
         Jinja2==2.10 \
         PyYAML==3.12 \
-        pyasn1==0.1.9 &&\
-    # Install toolchain
-    mkdir -p /root/bin/gcc-7-2018-q2-update &&\
-    curl --create-dirs -fsSLo /tmp/gcc-arm-none-eabi-linux.tar.bz2 'https://developer.arm.com/-/media/Files/downloads/gnu-rm/7-2018q2/gcc-arm-none-eabi-7-2018-q2-update-linux.tar.bz2?revision=bc2c96c0-14b5-4bb4-9f18-bceb4050fee7?product=GNU%20Arm%20Embedded%20Toolchain%20Downloads,64-bit,,Linux,7-2018-q2-update' &&\
-    tar -xf /tmp/gcc-arm-none-eabi-linux.tar.bz2 -C /root/bin/gcc-7-2018-q2-update --strip-components=1 &&\
-    # set paths
-    echo > ~/.bashrc &&\
-    echo 'export PATH=/root/bin/gcc-7-2018-q2-update/bin:$PATH' >> ~/.bashrc
```

Comment:
The lines that are being removed here were added in the previous commit, so instead fix the previous commit so it does not need to be fixed in a follow up commit.

Reply:
Done it in the latest push